pwa/src/services/getNameFromBijlageArray.ts

```diff
@@ -0,0 +1,8 @@
+export const getNameFromBijlageArray = (url: string, array: any[]) => {
+  if (!array) return;
+  const bijlage = array.find((option) => {
+    return option.URL_Bijlage === url;
+  });
+
+  return bijlage?.Titel_Bijlage;

```

Comment:
I think we might be able to shorthand this to, including adding a return type (which I think can also be undefined?):

```
export const getBijlageTitleFromURL = (url: string, bijlagen: any[]): string | undefined => {
  return bijlagen.find((bijlage) => bijlage.URL_Bijlage === url)?.Titel_Bijlage;
};
```